src/utility.js

```diff
@@ -392,3 +396,30 @@ function Benchmarker() {
   };
 };
 
+if (typeof exports !== 'undefined') {
+  exports.Benchmarker = Benchmarker;
+  exports.shouldAbort = shouldAbort;
+  exports.assert = assert;
+  exports.copy = copy;
+  exports.error = error;
+  exports.isJsLibraryConfigIdentifier = isJsLibraryConfigIdentifier;
+  exports.isJsOnlyIdentifier = isJsOnlyIdentifier;
+  exports.isNumber = isNumber;
+  exports.isPowerOfTwo = isPowerOfTwo;
+  exports.jsonCompare = jsonCompare;
+  exports.keys = keys;
+  exports.log2 = log2;
+  exports.mergeInto = mergeInto;
+  exports.range = range;
+  exports.safeQuote = safeQuote;
+  exports.set = set;
+  exports.setSize = setSize;
+  exports.setSub = setSub;
+  exports.setUnion = setUnion;
+  exports.sortedJsonCompare = sortedJsonCompare;
+  exports.splitter = splitter;
+  exports.stringifyWithFunctions = stringifyWithFunctions;
+  exports.unset = unset;
+  exports.warn = warn;
+  exports.warnOnce = warnOnce;
```

Comment:
this long list of exports is unfortunate. Is there no better way?

one option might be to define them all as
```javascript
exports = {
  set: ..
  setSize: ..
};
```
But I'm not sure that's nicer. Also if they are used in this file, you'd need `importAll` or such here as well.

Reply:
You can do this:

```
module.exports = {
  set,
  setSize,
}
```

Reply:
Thanks, I'll try that.